src/components/EmployeeModalForm/EmployeeCreateForm.tsx

```diff
@@ -49,41 +50,54 @@ export const EmployeeCreateForm: Component<Props> = ({ state, actions }) => {
             <FormInput
               name={firstName.name}
               value={firstName.value}
+              maxLength={INPUT_LENGTH_SHORT_MAX}
               type="text"
+              hasError={firstName.errors != null && firstName.errors.required}
               onInputChange={formActions.updateValue}
             />
+            {firstName.errors != null && firstName.errors.required && <FormHint theme="danger" label="First name is required" />}
           </FormField>
           <FormField labelText="Last Name" required={true}>
             <FormInput
               name={lastName.name}
               value={lastName.value}
+              maxLength={INPUT_LENGTH_SHORT_MAX}
               type="text"
+              hasError={lastName.errors != null && lastName.errors.required}
```

Comment:
i would change `hasError` to `errors` and pass it the entire `errors` object. The `errors` object will be undefined, if there are no errors.